src/test/java/it/gov/pagopa/reward/service/synchronous/op/CreateTrxSynchronousServiceImplTest.java

```diff
@@ -268,11 +270,11 @@ void authorizeTransactionAlreadyProcessed() {
             Assertions.fail("Expected an Exception");
 
         } catch (Exception e){
-            Assertions.assertInstanceOf(TransactionSynchronousException.class, e);
-            SynchronousTransactionResponseDTO resultResponse = ((TransactionSynchronousException) e).getResponse();
+            Assertions.assertInstanceOf(RewardCalculatorConflictException.class, e);
```

Comment:
catch RewardCalculatorConflictException instead of Exception and then asserting the type